mothership/test/src/org/labkey/test/tests/mothership/InProductMessagingTest.java

```diff
@@ -0,0 +1,133 @@
+package org.labkey.test.tests.mothership;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.pages.mothership.EditUpgradeMessagePage;
+import org.labkey.test.util.ExperimentalFeaturesHelper;
+import org.labkey.test.util.TestUser;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@Category({})
```

Comment:
Add to `Daily` suite
```suggestion
@Category({Daily.class})
```